lang/csharp/src/apache/main/Generic/GenericDatumReader.cs

```diff
@@ -87,7 +88,7 @@ protected override FixedAccess GetFixedAccess(FixedSchema readerSchema)
             return new GenericFixedAccess(readerSchema);
         }
 
-        class GenericEnumAccess : EnumAccess
+        private class GenericEnumAccess : EnumAccess
```

Comment:
By default methods are private (technically internal) unless otherwise set by an access modifier.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers
"Classes, records, and structs declared directly within a namespace (in other words, that aren't nested within other classes or structs) can be either public or internal. internal is the default if no access modifier is specified."